pkg/kine/drivers/sqlite/schema.go

```diff
@@ -13,10 +14,47 @@ import (
 // already
 
 const (
-	databaseSchemaMajorVersion = 1
-	databaseSchemaMinorVersion = 0
+	databaseSchemaMajorVersion int16 = 0
+	databaseSchemaMinorVersion int16 = 1
+	mask                       int32 = 0xFFFF // Mask for 2 bytes (16 bits)
 )
 
+type SchemaVersion int32
+
+var (
+	databaseSchemaVersion = ToSchemaVersion(databaseSchemaMajorVersion, databaseSchemaMinorVersion)
+)
+
+func ToSchemaVersion(major int16, minor int16) SchemaVersion {
```

Comment:
Can we call this `NewSchemaVersion` instead? The `ToXXX` doesn't have a subject. In my mind it would make sense if you already had an object so that you call `yyy.ToXXX` but in this case, you are creating a new one.